src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -86,6 +86,14 @@ public void Add(CookieCollection cookies)
             }
         }
 
+        public bool IsReadOnly
+        {
+            get
+            {
+                return false;
```

Comment:
@tijoytom can you please drive the decision here? Please close issue #12959 if we decide to ship .NET Core 1.2 this way. Thanks!